constants/index.ts

```diff
@@ -1,6 +1,8 @@
 import process from 'process'
 
 export { PROXY_LIST_API_URLS } from './PROXY_LIST_API_URLS'
+export { OCR_SPACE_PRO_API_USA } from './OCR_SPACE_PRO_API'
```

Comment:
Is there a reason to distinguish between the two (as opposed to keeping them in an array)?

Anyway, don't change it now because it works either way

Reply:
We don't use both of them. The USA server response slightly faster in the VPS. The second server is just in case.